### ISSUE TITLE
Old setting name used when using xdebug 3

### ISSUE THREAD

**Issue by kielabokkie** [pestphp/pest-intellij#239]
When running tests with coverage using xdebug 3 it always shows the following error in the output once all tests are done:

```
Xdebug: [Config] The setting 'xdebug.coverage_enable' has been renamed, see the upgrading guide at https://xdebug.org/docs/upgrade_guide#changed-xdebug.coverage_enable
```

I can see that this option is added here: https://github.com/pestphp/pest-intellij/blob/4469b6bfd75097536ae555bac828b2c5797858be/src/main/kotlin/com/pestphp/pest/coverage/PestCoverageProgramRunner.kt#L47

Not sure if it is possible to check the xdebug version and do something like `CoverageEngine.XDEBUG -> "xdebug.mode=coverage"` instead?

If you add the `xdebug.mode=coverage` setting in your xdebug ini file everything works fine, it's just the error that keeps on catching my eye.

**Comment by olivernybroe** [comment]
Hmm interesting, I'll add that in, thanks! 

It will be available in v1.3.0, which will be out in the next two days.

**Comment by kielabokkie** [comment]
@olivernybroe Thanks, that was fast! 🙌🏼
